### PR TITLE
Add prerequisite for Software Teams feature

### PR DESCRIPTION
**Pull request by jwitz**
Feel free to ignore the login docs, those changes have already been made to the main branch. Having some commit issues 😵

### CODE REVIEW

File: astro/log-in-to-astro.md [hunk 1]
```diff
@@ -5,9 +5,9 @@ id: log-in-to-astro
 description: Log in to Astro to access Astro features and functionality.
 ---
 
-You can use the Cloud UI and the Astro CLI to view and modify your workspaces, deployments, environment variables, tasks, and users. You need to authenticate your user credentials when you're using the Cloud UI or the Astro CLI for development on Astro.
```
Comment (jwitz) [inline astro/log-in-to-astro.md:8]: You can ignore this file, not sure why this change is being tracked in this PR

**Review by kwcanuck — APPROVED** [review]
